app/lib/core/services/service_locator.dart

```diff
@@ -1,26 +1,35 @@
 import 'package:app/core/services/database/database_service.dart';
+import 'package:app/core/view_models/add_history_view_model.dart';
 import 'package:app/core/view_models/atr_pre_view_model.dart';
 import 'package:app/core/view_models/nav_view_model.dart';
+import 'package:app/core/view_models/ongoing_view_model.dart';
 import 'package:app/core/view_models/signin_view_model.dart';
 import 'package:app/core/view_models/signout_view_model.dart';
+import 'package:app/core/view_models/signup_view_model.dart';
+import 'package:app/core/view_models/splash_screen_view_model.dart';
+import 'package:app/core/view_models/user_profile_view_model.dart';
 import 'package:app/core/view_models/welcome_view_model.dart';
 import 'package:get_it/get_it.dart';
-
 import 'authentication/auth_service.dart';
+import 'firestore/firestore_service.dart';
 import 'navigation/nav_service.dart';
 
 final locator = GetIt.instance;
 
 void setUpLocator() {
```

Comment:
I'm wondering is we should give this a better name and split the singletons from the factories (and add comments to tell people when to use what).

Reply:
We can definitely change the naming in the future